src/dotnet-install.sh

```diff
@@ -1188,6 +1189,11 @@ say "- The SDK installation doesn't need to persist across multiple CI runs."
 say "To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.\n"
 
 check_min_reqs
+if [ $? -ne 0 ]
```

Comment:
I'm not sure if this change is valid:
`set -e` stops script at any time non-zero code is returned from the statement. 'return 1' returns non-zero code.

Reply:
I didn't know of `set -e` and how it worked. Now I think this PR got too insignificant to merge. I will close it without merging unless you disagree.